docs/philosophy.md

```diff
@@ -30,6 +30,16 @@ As such, it is my conclusion that some moral fact-of-the-matter does not exist,
 
 Just because I think there is no such thing as morality, however, does not imply that I think moral statements are useless. They provide an important shorthand for conversations about how I would like people to act and how I would like society to function. As mentioned in the summary, there is a school of thought related to this called [Fictionalism](https://plato.stanford.edu/entries/fictionalism/#ViaEliAnt), which roughly equates to my beliefs on this. Without getting into the weeds too much, it claims that as long as moral statements reach their proposed goal (i.e. to make people act a certain way, to create agreement about how society should function, etc.) then we can treat them as if they are [truth-apt](https://www.oxfordreference.com/view/10.1093/oi/authority.20110803105953845) statements.
 
+## Maximizing One's Experience
+
+In the summary of my foundational beliefs, I talk about maximizing my own experience and helping others maximize theirs. While I stated that I generally leave this vague because different people have different ways to achieve this, I often run into a problem where people assume the most naive construction of this idea possible. It is assumed, especially when words like "hedonism" and "egoism" get used, that I conceptualize a moral world to be one where everyone just does whatever they want, be it murdering, stealing, etc. because it makes them happy. It is also assumed that I make no distinction between "lower and higher" pleasures. This is obviously a ridiculous position to hold.
+
+A thought experiment I often tell people is the following: You and four friends enter a room with five candy bars. You can either eat all of the candy bars because it would "maximize your experience," or you could share the candy equally. The naive construction of my belief would entail the former, but lets think about the consequences of this. My friends are now unhappy, they might not want to be friends with me anymore, next time they won't share with me, and really the outcome is in the long run (and potentially even immediately) I have certainly NOT maximized my experience. My friends being sad would make me sad, them not being friends with me anymore would be upsetting, you can imagine the rest.
```

Comment:
```suggestion
A thought experiment I often tell people is the following: You and four friends enter a room with five candy bars. You can either eat all of the candy bars because it would "maximize your experience," or you could share the candy equally. The naive construction of my belief would entail the former, but let's think about the consequences of this. My friends are now unhappy, they might not want to be friends with me anymore, next time they won't share with me, and really the outcome is in the long run (and potentially even immediately) I have certainly NOT maximized my experience. My friends being sad would make me sad, them not being friends with me anymore would be upsetting, you can imagine the rest.
```

Should be a contraction of "let" and "us".